filtered-resource-service/README.md

```diff
@@ -93,11 +123,12 @@ The Filtered-Resource Service requires a number of inputs from separate services
 
 ### Auditing within the Filtered Resource Service
 The Filtered Resource Service will send audit messages, specifically an `AuditErrorMessage` to the Audit Service via the error topic for the two following cases:
-1. No start marker was observed before reading the resources. If resources are processed by the Filtered Resource Service before a start marker is observed, 
-   it could indicate that there is an issue earlier on in the system which could cause the messages to fall out of ordered. In this case, an Audit Error Message is created, containing a `NoStartMarkerObserved` exception, 
-   which is then sent to the Audit Service, and finally, the processing of the request is stopped. 
-2. No Resources were contained in the request. If a start marker is observed, but no resources are contained in the request, the request could be invalid, for resources that aren't known to Paliasde, 
-   or for resources that have been redacted. In this case, an Audit Error Message is created, containing a `NoResourcesObserved` exception, and is then sent to the Audit Service. 
+1. No start marker was observed before reading the resources. 
+   If resources are processed by the Filtered Resource Service before a start marker is observed, it could indicate that there is an issue earlier on in the system which could cause the messages to fall out of order. 
```

Comment:
```suggestion
   If resources are processed by the Filtered-Resource Service before a start marker is observed, it could indicate that there is an issue earlier on in the system which could cause the messages to fall out of order. 
```